db/migrate/20230817233805_create_posts.rb

```diff
@@ -1,9 +1,15 @@
-class CreateComments < ActiveRecord::Migration[7.0]
+# frozen_string_literal: true
+
+class CreatePosts < ActiveRecord::Migration[7.0]
   def change
-    create_table :comments do |t|
-      t.references :author, foreign_key: { to_table: :users }
-      t.integer :post, foreign_key: { to_table: :posts }
+    create_table :posts do |t|
+      t.references :author, null: false, foreign_key: true
+      t.string :title
       t.text :text
+      t.datetime :created_at
+      t.datetime :updated_at
+      t.integer :comments_counter
+      t.integer :likes_counter
 
       t.timestamps
```

Comment:
- [ ] Great job so far 👏🏻 However there are a few things you need to address  before your project is approved:
-  1 `t.references :author, null: false, foreign_key: true` The  Associations between models must be configured correctly according to the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png). Since we are using the `author_id` column to reference the modal user, we need to show that the `'author_id'` references the User class. to do this, you need to update the value of the 'foreign_key' attribute to `{ to_table: :users }`. You can write the query like this: 


```ruby
class AddUserRefToPosts < ActiveRecord::Migration[7.0]
  def change
    add_reference :posts, :author, foreign_key: { to_table: 'users' } # this will create author_id and a corresponding index
  end
end
```

- You are not obliged to add those two columns : 
```Ruby
      t.datetime :created_at
      t.datetime :updated_at
```
- Since you have ` t.timestamps` this will make them get generated automatically based on the actual creation time and updated time